kits/sparktrace/prompts/sparktrace-analyst_generate-json_user.md

```
@@ -0,0 +1,13 @@
+Symptom:
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)